scene/3d/mesh_instance.cpp

```diff
@@ -96,7 +96,7 @@ void MeshInstance::_get_property_list(List<PropertyInfo> *p_list) const {
 	ls.sort();
 
 	for (List<String>::Element *E = ls.front(); E; E = E->next()) {
-		p_list->push_back(PropertyInfo(Variant::REAL, E->get(), PROPERTY_HINT_RANGE, "0,1,0.01"));
+		p_list->push_back(PropertyInfo(Variant::REAL, E->get(), PROPERTY_HINT_RANGE, "0,1,0.00001"));
```

Comment:
Looks like something that should go in its own PR.

Reply:
i would change it to
"0,1,0.001,or_lesser" so it does not look like ass in the inspector but you can still use tiny values.

Reply:
As mentioned in another comment, `or_lesser` does not imply that you can use finer precision, only that you can go below the min (0), which would be wrong here.

In the current version, 5 digits of precision seems overkill, no? It was 4 digits originally, do you really need 5?

Reply:
It was to match:

```
@@ -4131,7 +4131,7 @@ void RasterizerStorageGLES3::mesh_render_blend_shapes(Surface *s, const float *p
	for (int ti = 0; ti < mtc; ti++) {
		float weight = p_weights[ti];
 
		if (weight < 0.001) //not bother with this one
		if (weight < 0.00001) //not bother with this one
			continue;
 
		glBindVertexArray(s->blend_shapes[ti].array_id);
```

Reply:
Hm ok, makes sense.